Source/DafnyDriver/TextLogger.cs

```diff
@@ -13,13 +14,14 @@ public class TextLogger {
   }
 
   public void LogResults(List<(Implementation, VerificationResult)> verificationResults) {
```

Comment:
`VerificationResult` also contains a (private) `Implementation` field, so consider exposing that so you can replace `(Implementation, VerificationResult)` with `VerificationResult`

Reply:
It'll require a Boogie change, though, so I think perhaps it's a refactor best left to a later PR.